monitoring/base/grafana-operator/dashboards/all-nodes.yaml

```
@@ -719,7 +719,7 @@ spec:
                 "steppedLine": false,
                 "targets": [
                     {
-                        "expr": "deriv(time() - node_boot_time_seconds{}) < bool 0.1",
+                        "expr": "changes(node_boot_time_seconds{}[15m:]) > 0",
```

I didn't know the notation of `[15m:]`.
Are you familiar with this type of query? Or did you constructed with the help of autocomplete?

```suggestion
                        "expr": "changes(node_boot_time_seconds[15m]) > 0",
```

---

@binoue 
> <resolution> is optional, which is equivalent to step in /query_range API.

It seems optional, isn't it?

---

Ah, you mean ':' may not be optional.

---

@morimoto-cybozu 

Sorry. I was misunderstanding.
The resolution rate was for aggregating data with the lower resolution/range.

See [this official doc](https://prometheus.io/blog/2019/01/28/subquery-support/#motivation).
In this case, the range vector is not the aggregated data, so no need to specify the resolution.(i.e. no colon)